<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>734</width>
    <height>499</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_29">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tab_7">
       <attribute name="title">
        <string>Config</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_9">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_10">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_21">
            <item>
             <widget class="QLabel" name="label_23">
              <property name="font">
               <font>
                <pointsize>11</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Port</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="comboBox1"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_22">
            <item>
             <widget class="QLabel" name="label_24">
              <property name="text">
               <string>Baud Rate</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit3"/>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_7">
              <property name="text">
               <string>Apply</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="Line" name="line_7">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_23">
            <item>
             <widget class="QLabel" name="label_25">
              <property name="text">
               <string>No. Ports</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit1">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="autoFillBackground">
               <bool>false</bool>
              </property>
              <property name="frame">
               <bool>true</bool>
              </property>
              <property name="readOnly">
               <bool>false</bool>
              </property>
              <property name="clearButtonEnabled">
               <bool>false</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_24">
            <item>
             <widget class="QLabel" name="label_26">
              <property name="text">
               <string>Vendor ID</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit2">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="autoFillBackground">
               <bool>false</bool>
              </property>
              <property name="frame">
               <bool>true</bool>
              </property>
              <property name="readOnly">
               <bool>false</bool>
              </property>
              <property name="clearButtonEnabled">
               <bool>false</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_25">
            <item>
             <widget class="QLabel" name="label_27">
              <property name="text">
               <string>Product ID</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit5">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="autoFillBackground">
               <bool>false</bool>
              </property>
              <property name="frame">
               <bool>true</bool>
              </property>
              <property name="readOnly">
               <bool>false</bool>
              </property>
              <property name="clearButtonEnabled">
               <bool>false</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_26">
            <item>
             <widget class="QLabel" name="label_28">
              <property name="text">
               <string>Baud Rate</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit4">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="autoFillBackground">
               <bool>false</bool>
              </property>
              <property name="frame">
               <bool>true</bool>
              </property>
              <property name="readOnly">
               <bool>false</bool>
              </property>
              <property name="clearButtonEnabled">
               <bool>false</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_8">
       <attribute name="title">
        <string>Control</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0" rowspan="2" colspan="2">
         <widget class="Line" name="line_8">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </item>
        <item row="0" column="2" rowspan="6">
         <widget class="QCustomPlot" name="Grafica" native="true"/>
        </item>
        <item row="1" column="1">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLCDNumber" name="lcdNumber_3"/>
        </item>
        <item row="3" column="1">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Tiempo de muestreo (ms)</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QLCDNumber" name="lcdNumber_2"/>
        </item>
        <item row="5" column="1">
         <widget class="QSlider" name="verticalSlider">
          <property name="maximum">
           <number>10</number>
          </property>
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </item>
        <item row="6" column="2">
         <widget class="QProgressBar" name="progressBar">
          <property name="maximum">
           <number>1024</number>
          </property>
          <property name="value">
           <number>24</number>
          </property>
         </widget>
        </item>
        <item row="6" column="3">
         <widget class="QLabel" name="label_29">
          <property name="text">
           <string>ADC:</string>
          </property>
         </widget>
        </item>
        <item row="6" column="4">
         <widget class="QLabel" name="label_30">
          <property name="text">
           <string>1</string>
          </property>
         </widget>
        </item>
        <item row="6" column="5">
         <widget class="QLCDNumber" name="lcdNumber"/>
        </item>
        <item row="7" column="0" colspan="2">
         <widget class="QPushButton" name="pushButton_9">
          <property name="text">
           <string>Terminar comunicacion</string>
          </property>
         </widget>
        </item>
        <item row="7" column="2">
         <widget class="QProgressBar" name="progressBar_4">
          <property name="maximum">
           <number>1024</number>
          </property>
          <property name="value">
           <number>24</number>
          </property>
         </widget>
        </item>
        <item row="7" column="3">
         <widget class="QLabel" name="label_47">
          <property name="text">
           <string>ADC:</string>
          </property>
         </widget>
        </item>
        <item row="7" column="4">
         <widget class="QLabel" name="label_48">
          <property name="text">
           <string>2</string>
          </property>
         </widget>
        </item>
        <item row="7" column="5">
         <widget class="QLCDNumber" name="lcdNumber_10"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>734</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuaaa">
    <property name="title">
     <string>aaa</string>
    </property>
   </widget>
   <addaction name="menuaaa"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>../../qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
